packages/Panel/stories/Panel.backyard.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { getStoryName } from "storybook/storyTree";
+import { testStoryParameters } from "storybook/assets/storyParameters";
+import ExampleStory from "storybook/components/ExampleStory";
+import WithSortable from "./sandbox/WithSortable";
+import Panel from "../src";
+
+const storyName = getStoryName("Panel");
+
+export default {
+  title: `${storyName}/Backyard/Sandbox`,
+  component: Panel,
```

Comment:
Do you know what this line does? It seems to work the same without it and the requisite `import`.

Reply:
The component field is optional (but encouraged!), and is used by addons for automatic prop table generation and display of other component metadata. title should be unique, i.e. not re-used across files.

This is from the docs